packages/react-router/src/route.ts

```diff
@@ -466,6 +466,15 @@ export interface UpdatableRouteOptions<
     meta?: AnyRouteMatch['meta']
   }
   ssr?: boolean
+  codeSplitGroupings?: Array<
```

Comment:
this must be an array of string literals right? otherwise the AST traversal will be hard / impossible

if my assumption is correct, is this documented? do we throw a descriptive error in case this is not using string literals?